src/v/model/tests/randoms.h

```diff
@@ -98,16 +99,19 @@ inline model::broker_properties random_broker_properties() {
           random_generators::gen_alphanum_string(
             random_generators::get_int(1, 100)));
     }
+    uint32_t memory_gb = random_generators::get_int<uint32_t>(1, 100);
```

Comment:
We should generate a value in bytes and covert to GB, not the other way around. It's more general and would catch any bugs related to bytes != gb * 2^30 (which will be the usual case in the real world I think).